packages/@ember/engine/index.ts

```diff
@@ -467,7 +467,8 @@ function resolverFor(namespace: Engine) {
   return ResolverClass.create(props);
 }
 
-function buildInitializerMethod<
+/** @internal */
+export function buildInitializerMethod<
```

Comment:
Per comments above, we should update this to be smarter about how it does its thing.